scripts/process.py

```diff
@@ -1,12 +1,13 @@
-
+"""
+    This script processes the archive folder and puts processed data to data/ folder.
+    Note:
+    1. Currently, the FTP server doesn't produce the data for the CSIRO_Recons_gmsl_mo_{year}.csv file, it only produces
+       yearly data. So, in order to not mess the process of the data, old data from CHURCH/WHITE source for 2015 monthly is used.
+"""
+import os
 import csv
 import zipfile
-
-import os
-import urllib
-#import dataconverters.xls
-epa_src = "http://www3.epa.gov/climatechange/images/\
```

Comment:
@gradedSystem are we saying that EPA source is not valid anymore? Did they stop providing data?

Reply:
It is a valid source however data only available until 2014 and the original source now deprecated and this one is currently much newer: www.cmar.csiro.au